_layouts/default.html

```diff
@@ -16,7 +16,7 @@
 
     <div class="page-content">
       <div class="wrapper">
-        {{ content }}
+        {{ content | replace: '<a href="https://knapsackpro.com', '<a rel="noopener noreferrer" target="_blank" href="https://knapsackpro.com' | replace: '<a href="https://docs.knapsackpro.com', '<a rel="noopener noreferrer" target="_blank" href="https://docs.knapsackpro.com' | replace: '<a href="http', '<a rel="nofollow noopener noreferrer" target="_blank" href="http' }}
```

Comment:
This seems fragile (are all the links to replace starting with the first attribute being `href`?). The alternatives would be:
- Use a regexp like /(href=\.+knapsackpro.com/rel="noopener noreferrer" target="_blank" $1/g (pseudo-code)
- More sophisticated logic with a parser

But I'm fine leaning on the fragile side in this case. Go with whatever makes the most sense.

Reply:
I'll keep it as it is. It's just simple replace (no regexp supported):
https://jekyllrb.com/docs/liquid/filters/
https://shopify.github.io/liquid/filters/replace/